src/functions-and-arrays.js

```diff
@@ -1,41 +1,122 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  }
+  else {
+    return num2;
+  }
+}
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
-
+function findLongestWord(words) {
+  let longestWord = ' ';
+  for ( let i = 0; i < words.length; i++){
+    if(words[i].length > longestWord){
+	longestWord = words[i];
+     }
+  return longestWord;
+  }
+} 
```

Comment:
wie du sicherlich mittlerweile weißt, hast du hier und da die if Bedingung, die "null" zurückgeben soll, wenn das Array leer ist, nicht ergänzt. Das stand auch nicht in der Lab selber, sondern war eher dafür da um zu sehen, ob ihr mit den Angaben im Test gut umgehen könnt

Reply:
und du musst das return longestWord außerhalb des for loops machen

Reply:
und es muss heißen:
`if(words[i].length > longestWord.length){`